skt/runner.py

```diff
@@ -496,16 +500,22 @@ def run(self, url, release, wait=False, host=None, uid="",
             hostname = "(%s) " % host
             hostnametag = '<hostname op="=" value="%s"/>' % host
 
-        jobid = self.__jobsubmit(self.__getxml({'KVER': release,
-                                                'KPKG_URL': url,
-                                                'UID': uid,
-                                                'ARCH': arch,
-                                                'HOSTNAME': hostname,
-                                                'HOSTNAMETAG': hostnametag}))
-
-        if wait:
-            self.wait(jobid, reschedule)
-            ret = self.__getresults()
+        try:
+            jobid = self.__jobsubmit(self.__getxml(
+                {'KVER': release,
+                 'KPKG_URL': url,
+                 'UID': uid,
+                 'ARCH': arch,
+                 'HOSTNAME': hostname,
+                 'HOSTNAMETAG': hostnametag}
+            ))
+
+            if wait:
+                self.wait(jobid, reschedule)
+                ret = self.__getresults()
+        except Exception as exc:
+            logging.error(exc)
+            ret = 2
```

Comment:
Could you please document the function, including the exit status?

Reply:
Added!